grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1083,6 +1085,10 @@ class StockMovementService {
     }
 
     void createMissingPicklistForStockMovementItem(StockMovementItem stockMovementItem) {
+        if (stockMovementItem?.requisitionItem?.requisition?.status < RequisitionStatus.PICKING) {
```

Comment:
I agree with this change, but I'm worried about unintended consequences here so we'll need to test this thoroughly.

Reply:
It's already tested by Katarzyna and UATed by Kelsey